lib/kitto/router.ex

```diff
@@ -21,6 +21,20 @@ defmodule Kitto.Router do
     end
   end
 
+  post "dashboards" do
+    {:ok, body, conn} = read_body conn
+    command = body |> Poison.decode! |> Map.put_new("dashboard", "*")
+    Kitto.Notifier.broadcast! "_kitto", command
+    conn |> send_resp(204, "")
+  end
+
+  post "dashboards/:id" do
+    {:ok, body, conn} = read_body conn
+    command = body |> Poison.decode! |> Map.put("dashboard", id)
```

Comment:
Do you want to open a separate issue for the tech debt to clean up the use of `Poison.decode!`?